src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  --color-accent: #00acdc;
+  --color-default: #000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
```

Comment:
The 'flex-direction: row;' property is unnecessary because it is the default value for flex containers. Remove it to simplify the code.

Reply:
done